crates/nu-command/src/experimental/is_admin.rs

```diff
@@ -48,3 +47,55 @@ impl Command for IsAdmin {
         ]
     }
 }
+
+/// Returns `true` if user is root; `false` otherwise
+fn is_root() -> bool {
+    is_root_impl()
+}
+
+#[cfg(unix)]
+fn is_root_impl() -> bool {
+    nix::unistd::Uid::current().is_root()
+}
+
+#[cfg(windows)]
+fn is_root_impl() -> bool {
+    use windows::Win32::{
+        Foundation::{CloseHandle, HANDLE},
+        Security::{GetTokenInformation, TokenElevation, TOKEN_ELEVATION, TOKEN_QUERY},
+        System::Threading::{GetCurrentProcess, OpenProcessToken},
+    };
+
+    let mut handle = HANDLE::default();
+    let mut elevated = false;
+
+    // Checks whether the access token associated with the current process has elevated privileges.
+    unsafe {
+        // Opens the access token associated with the current process.
+        if OpenProcessToken(GetCurrentProcess(), TOKEN_QUERY, &mut handle).as_bool() {
+            let mut elevation = TOKEN_ELEVATION::default();
+            let mut size = std::mem::size_of::<TOKEN_ELEVATION>() as u32;
+
+            // Retrieves elevation token information about the access token associated with the current process.
```

Comment:
```suggestion
            // Retrieves elevation token information about the access token associated with the current process.
            // Call available since XP
            // https://learn.microsoft.com/en-us/windows/win32/api/securitybaseapi/nf-securitybaseapi-gettokeninformation
```